HeartbeatLogging/Tests/Unit/HeartbeatsBundleTests.swift

```diff
@@ -0,0 +1,322 @@
+// Copyright 2021 Google LLC
```

Comment:
This file is a rename of HeartbeatLogging/Tests/Unit/HeartbeatInfoTests.swift with added tests for new remove API.

Reply:
I used `git mv` in 5d8f6db to keep its history but it didn't seem to work.